CDCgov/NEDSS-Modernization
# Cnft2 952/download page library pdf

## Description
Enables downloading pdf of the page library from the manage pages table.


## Tickets

https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT2/boards/99?modal=detail&selectedIssue=CNFT2-952&assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4
## Steps to verify

1. Navigate to /page-builder/manage/pages
2. Click the print icon near the top.
3. verify that pagesLibrary.pdf is downloaded and viewable.
4. Run 'npm run test' and verify all tests pass
